__tests__/app.test.js

```diff
@@ -152,4 +152,43 @@ describe('GET /api/articles/:article_id/comments', () => {
 
 	})
 })
-
+describe.only('PATCH /api/articles/:article_id', () => {
+	it('Should be able to PATCH an article to change the value of its votes and respond with that article and the way it looks after', () => {
+		const patchObj = { inc_votes: 3}
+		return request(app)
+		.patch('/api/articles/1')
+		.send(patchObj)
+		.expect(200)
+		.then( (response) => {
+			expect(response.body.votes).toBe(103)
+		})
+	})
+	it('Should not allow the user to increment the votes of an article that isnt there', () => {
```

Comment:
Ideally want to check the correct messages are being sent back for each of these as well